doc/src/docs/EngineeringReference/13b-EncyclopaedicRefs.md

```diff
@@ -1811,6 +1811,8 @@ This model simulates the thermal performance of the indoor DX cooling coil, and
 
 When the model determines performance at Speed 1 (the lowest speed) or cycling between OFF and Speed 1, its performance is almost the same as the performance for the Coil:Cooling:DX:SingleSpeed model. However, the outlet conditions are calculated slightly differently. Therefore, the Coil:Cooling:DX:SingleSpeed model may be considered as a subset of the model described here. When the multispeed coil model determines performance at higher speeds (above 1), the model linearly interpolates the performance at two consecutive speeds (n-1 and n) as needed to meet the cooling load, with the fraction of time at each speed established by the speed ratio.
 
+When single mode operation is specified at higher speeds (above 1), defined in the UnitarySystemPerformance:Multispeed object, a child object of AirLoopHVAC:UnitarySystem, its performance is almost the same as the performance for the Coil:Cooling:DX:SingleSpeed model at different flow rate and capacity with given speed number. No liner interpolation is performed between two adjacent speeds.
```

Comment:
Good catch for typos. Your suggestion is added. I would like to add one more sentence. "This operation does not apply to speed 1 operation and the case when the sensible load is above the highest sensible capacity."